<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegWin</class>
 <widget class="QMainWindow" name="RegWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>708</width>
    <height>449</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Регистрация</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.jpg</normaloff>icon.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="3">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="0">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1" colspan="2">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="4" column="0">
         <widget class="QLabel" name="label_5">
          <property name="styleSheet">
           <string notr="true">font: 14pt &quot;Calibri&quot;;</string>
          </property>
          <property name="text">
           <string>Пароль</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="styleSheet">
           <string notr="true">font: 14pt &quot;Calibri&quot;;</string>
          </property>
          <property name="text">
           <string>Дата рождения</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="styleSheet">
           <string notr="true">font: 14pt &quot;Calibri&quot;;</string>
          </property>
          <property name="text">
           <string>Номер подразделения</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="otdel_num">
          <property name="styleSheet">
           <string notr="true">font: 14pt &quot;Calibri&quot;;</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="styleSheet">
           <string notr="true">font: 14pt &quot;Calibri&quot;;</string>
          </property>
          <property name="text">
           <string>ФИО</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="password_value">
          <property name="font">
           <font>
            <family>Calibri</family>
            <pointsize>14</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;пароль должен быть длинной не менее 8 символов и содержать заглавные и строчные буквы, цифры и специальные символы&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="toolTipDuration">
           <number>10000</number>
          </property>
          <property name="statusTip">
           <string/>
          </property>
          <property name="whatsThis">
           <string/>
          </property>
          <property name="accessibleDescription">
           <string/>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 14pt &quot;Calibri&quot;;</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="styleSheet">
           <string notr="true">font: 14pt &quot;Calibri&quot;;</string>
          </property>
          <property name="text">
           <string>Логин</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="name">
          <property name="styleSheet">
           <string notr="true">font: 14pt &quot;Calibri&quot;;</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="username_value">
          <property name="statusTip">
           <string/>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 14pt &quot;Calibri&quot;;</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDateEdit" name="birth_date">
          <property name="styleSheet">
           <string notr="true">font: 14pt &quot;Calibri&quot;;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="check_data">
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;Calibri&quot;;</string>
        </property>
        <property name="text">
         <string>Я согласен(а) на предоставление персональных данных приложению</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="check_rules">
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;Calibri&quot;;</string>
        </property>
        <property name="text">
         <string>Я согласен(а) с правилами пользования приложением</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>286</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="pushButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Calibri&quot;;</string>
      </property>
      <property name="text">
       <string>Зарегистрироваться</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
